moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -125,7 +125,19 @@ class Servo
    * @param The last commanded joint states.
    * @return The next state stepping towards the required halting state.
    */
-  std::pair<bool, KinematicState> smoothHalt(const KinematicState& halt_state) const;
+  std::pair<bool, KinematicState> smoothHalt(const KinematicState& halt_state);
+
+  /**
+   * \brief Updates the smoother, if one exists, by filtering with a new state.
```

Comment:
Maybe reword this to emphasize that this is applying smoothing to `state` and updating it accordingly.